qemu/tests/qemu_guest_agent.py

```diff
@@ -2554,6 +2556,8 @@ def gagent_check_fsinfo(self, test, params, env):
                                   mount_pt, logging.info)
             fs_type_qga = fs["type"]
             cmd_get_disk = params["cmd_get_disk"] % mount_pt.replace("/", r"\/")
+            if params["os_type"] == "windows":
+                cmd_get_disk = params["cmd_get_disk"] % device_id.replace("\\", r"\\")
```

Comment:
Could you explain why you have this replacement?

Reply:
you mean explain in script? 
this replacement because of the DeviceID just need to use this way '\\\\?\\Volume{xxx}\\',or we cannot get correctly value.

Reply:
@6-dehan No you don't need to comment in script, just I want to understand the detail. Now it's clear, thanks.